README.md

```diff
@@ -105,6 +106,27 @@ Table of contents:
 * [`#ng-2 Slack Channel`](https://angularbuddies.slack.com/messages/ng-2/) on AngularBuddies ([Sign up](http://www.angularbuddies.com/))
 * [`#angular2 Slack Channel`](https://dartlang.slack.com/messages/angular2) on dartlang.slack.com ([Sign up](https://dartlang-slack.herokuapp.com/))
 
+### Angular Google Developer Experts
```

Comment:
I think you should use `####` and there seems to already be a section of `Experts on Twitter` right under.